src/routes/tutorials/transfer-from-ethereum.svelte

```diff
@@ -26,104 +26,56 @@
     <h2>Transfer from Ethereum</h2>
 
     <p>
-      If you have Ethereum (ETH) or Ethereum ERC-20 tokens, you can use the Avalanche-Ethereum bridge to bring them to
-      Avalanche and use Pangolin. The bridge allows you to transfer assets between both blockchain networks. The diagram
-      below illustrates how the assets flow from Ethereum to Avalanche to use for Pangolin.
+      If you have Wrapped Ether (WETH) or Ethereum ERC-20 tokens, you can use the <a href="https://bridge.avax.network/" target="_blank">Avalanche Bridge</a> to bring them to
+      Avalanche and use Pangolin. The bridge allows you to transfer assets between both blockchain networks.
     </p>
 
-    <img src="/tutorials/transfer-from-ethereum/001.png" />
+    <iframe width="560" height="315" src="https://www.youtube.com/embed/RLnNMfINwS0" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
     <h3 class="flex whitespace-pre-wrap group" id="transfer-eth">
       <a
         href="#transfer-eth"
         class="absolute no-underline opacity-0 group-hover:opacity-100 after:hash"
         aria-label="Anchor"
         style="margin-left: -1em; padding-right: 0.5em; box-shadow: none; color: rgb(161, 161, 170);"></a><span
-        >Transferring Ethereum (ETH) to Avalanche</span
+        >Transferring Ethereum ERC-20 Tokens to Avalanche</span
       >
     </h3>
 
     <ol>
       <li>
-        Access the bridge portal <a href="https://aeb.xyz/" target="_blank" class="hover:no-underline">here</a>
+        Access the <a href="https://bridge.avax.network/" target="_blank">Avalanche Bridge</a>
       </li>
       <li>
-        Select "Wrap Token"
-        <img src="/tutorials/transfer-from-ethereum/002.png" />
+        Select "Metamask" and login into your wallet
+        <img src="/tutorials/transfer-from-ethereum/010.png" />
```

Comment:
Can we change the image number to the order they're on the page? `001.png`, `002.png` etc etc

Reply:
Updated!